prowler/providers/aws/services/codebuild/codebuild_project_no_plaintext_credentials/codebuild_project_no_plaintext_credentials.py

```diff
@@ -0,0 +1,23 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.codebuild.codebuild_client import codebuild_client
+
+
+class codebuild_project_no_plaintext_credentials(Check):
+    def execute(self):
+        findings = []
+        for project in codebuild_client.projects.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = project.region
+            report.resource_id = project.name
+            report.resource_arn = project.arn
+            report.status = "PASS"
+            report.status_extended = f"CodeBuild project {project.name} does not have sensitive environment plaintext credentials."
+            sensitive_vars = ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"]
```

Comment:
What do you think about using the `audit_config` to allow users to add/remove sensitive environment variables? It'd be a great addition!

Reply:
Good point! I've just added it.